ckanext/dia_theme/fanstatic/rua/css/rua.css

```diff
@@ -4548,7 +4649,7 @@ td {
   .site-footer__copyright {
     background-color: #fff;
     padding: 2.375rem 0; }
-  @media screen and (min-width: 1200px) {
+  @media screen and (min-width: 1100px) {
```

Comment:
Have we chosen 1100px because the menu has a lot of items?

Reply:
Yeah any less than 1100 and the header nav menu items start wrapping